corehq/apps/case_importer/tasks.py

```diff
@@ -40,7 +42,8 @@ def bulk_import_async(config, domain, excel_id):
     try:
         case_upload.check_file()
     except ImporterError as e:
-        return {'errors': get_importer_error_message(e)}
+        update_task_state(bulk_import_async, states.FAILURE, {'errors': get_importer_error_message(e)})
+        raise Ignore()
```

Comment:
Pulled this process from https://github.com/dimagi/commcare-hq/blob/3355a1fcf0d8a788375f3f72abc1a7ec0566aaa8/corehq/apps/hqmedia/tasks.py#L223